modules/ROOT/Edits/endpoint-ftp-send.adoc

```diff
@@ -0,0 +1,68 @@
+= FTP Send Endpoint Settings
+
+FTP send endpoint settings configure where to send messages in the following contexts:
+
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners
+
+* Target at host
++
+Target endpoint in an inbound message flow to which translated application JSON and XML messages are sent to the backend systems
+
+A host, partner, or 3rd party connection can own FTP send endpoints.
+
+Create FTP send endpoints by configuring these settings on the Host or Partner pages:
+
+[%header%autowidth.spread]
+|===
+|Name |Description | Required
+
+| Owner
+| The trading partner or host that owns the endpoint.
+| Yes
+
+| Protocol
+| The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
+| Yes
+
+|Endpoint Name
+| The name used to identify the endpoint. The value in the *Endpoint Name* field is generated automatically, but can be modified manually.
+| Yes
+
+|Description
+|The user-defined field for describing the purpose and use of the endpoint.
```

Comment:
Slightly different wording than used in earlier "purpose and use" texts; standardize repeated texts.